drupal/sites/all/features/user_profiles/user_profiles.features.content.inc

```diff
@@ -705,13 +705,13 @@ Zimbabwe|Zimbabwe',
 
   // Translatables
   // Included for use with string extractors like potx.
-  t('Avatar');
-  t('Country');
-  t('Profile image');
-  t('Website URL');
-  t('Your opinions about this project');
-  t('Your personal background');
-  t('Zip code');
+  bts('Avatar', array(), NULL, 'boinc:account-profile-edit');
+  bts('Country', array(), NULL, 'boinc:account-profile-edit');
+  bts('Profile image', array(), NULL, 'boinc:account-profile-edit');
+  bts('Website URL', array(), NULL, 'boinc:account-profile-edit');
+  bts('Your opinions about this project', array(), NULL, 'boinc:account-profile-edit');
+  bts('Your personal background', array(), NULL, 'boinc:account-profile-edit');
+  bts('Postal code', array(), NULL, 'boinc:account-profile-edit');
```

Comment:
This looks inconsistent. The labels 'Your personal background' and 'Your opinions about this project' are wrapped with `bts()` here and and in lines 63 and 493 above while 'Postal code' is only wrapped in `bts()` here put not on line 698. I don't know whats the right way but it should be consistent I guess.